website/docs/d/dl_gateway.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+
+layout: "ibm"
+page_title: "IBM : Direct Link gateway Virtual Connections"
+sidebar_current: "docs-ibm-datasources-dl-gateway"
+description: |-
+Manages IBM Cloud Infrastructure Directlink gatway's Virtual Connections.
+
+---
+
+# ibm\_dl_virtual_connections
+
+Import the details of an existing IBM Cloud Infrastructure directlink gateway's virtual connections as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
Import the details of an existing IBM Cloud Infrastructure directlink gateway and its virtual connections